airbyte-integrations/connectors/source-google-sheets/integration_tests/expected_records.jsonl

```diff
@@ -0,0 +1,4033 @@
+{"stream":"Sheet6-4000-rows","data":{"ID":"2630","Name":"ZLjnWxcdP"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2631","Name":"GBXPcMhHe"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2632","Name":"XCrvpWRZD"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2633","Name":"ITRgzNSmT"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2634","Name":"iQcLfPrNK"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2635","Name":"vTbcEPEnu"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2636","Name":"CluKgYzWC"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2637","Name":"QwKMClkRS"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2638","Name":"jVSiTkiOg"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2639","Name":"dtiDfaDiB"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2640","Name":"xYpArrCbv"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2641","Name":"uJeEJDuUN"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2642","Name":"MrmYWAADB"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2643","Name":"fpRLFfabc"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2644","Name":"uHnSghMSw"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2645","Name":"mYIGcBBop"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2646","Name":"NviSEGoCu"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2647","Name":"HQnEeWRhq"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2648","Name":"nunnOjklZ"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2649","Name":"iBkFywzCJ"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2650","Name":"SFBLaKrOg"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2651","Name":"QQLZECMUO"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2652","Name":"wXHtgETIH"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2653","Name":"QFFWQVxNW"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2654","Name":"VLTTHBrSA"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2655","Name":"zoTgMCBpG"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2656","Name":"dJkUxyHcJ"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2657","Name":"WsDCfjVjR"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2658","Name":"IRqDptXBg"},"emitted_at":1673989572000}
+{"stream":"Sheet6-4000-rows","data":{"ID":"2659","Name":"QmmhfvOaQ"},"emitted_at":1673989572000}
+{"stream"
```

Comment:
Do we really need the 4033 sam-structure records? Can we leave 1? Could we also use *.`txt` instead of `*.jsonl`?